tests/cypress/latest/support/commands.ts

```diff
@@ -39,23 +39,30 @@ Cypress.Commands.add('namespaceAutoImport', (mode) => {
   cy.reload(true);
   cy.getBySel('sortable-table-0-action-button').click();
 
-  cy.contains(mode + ' CAPI Auto-Import')
-    .click();
+
+  // If the desired mode is already in place, then simply reload the page.
+  cy.get('.list-unstyled.menu').then(($list) => {
+    if ($list.text().includes(mode + ' CAPI Auto-Import')) {
+      cy.contains(mode + ' CAPI Auto-Import').click();
+    } else {
+      // Workaround to close the dropdown menu
+      cy.reload();
+    }
+  })
   cy.namespaceReset();
 });
 
 // Command to set namespace selection
+// TOOD(pvala): Could be improved to check if the namespace is already set before changing it
 Cypress.Commands.add('setNamespace', (namespace) => {
-  cy.contains('Only User Namespaces') // eslint-disable-line cypress/unsafe-to-chain-command
-    .click()
-    .type(namespace + '{enter}{esc}');
+  cy.get('.ns-dropdown', { timeout: 12000 }).trigger('click');
```

Comment:
```suggestion
  cy.getBySel('namespaces-dropdown').click();
```
Since cypress best practice to use _data-testid_, if available